includes/Checker/Stable_Check.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Trait WordPress\Plugin_Check\Checker\Stable_Check
```

Comment:
Shouldn't these traits live in `WordPress\Plugin_Check\Traits`?

Reply:
@felixarntz yes, I've moved these now